ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleManager.java

```diff
@@ -81,19 +92,18 @@ private void cleanLeakedBannerAdapters() {
     }
   }
 
-  // TODO: Make this method return an AdError object instead of a boolean.
-  synchronized boolean canRequestBannerAd(@NonNull String placementId,
+  synchronized AdError canRequestBannerAd(@NonNull String placementId,
       @Nullable String requestUniqueId) {
     cleanLeakedBannerAdapters();
 
     VungleBannerAd bannerAd = mVungleBanners.get(placementId);
     if (bannerAd == null) {
-      return true;
+      return null;
```

Comment:
@HuFangshuai should we return AdRequest.INTERNAL_ERROR in these null cases?

Reply:
Null value here means we can go ahead to start loading ads. Refer to: https://github.com/Vungle/googleads-mobile-android-mediation/pull/103/files#diff-e2293cecb485c28b1c1574747eb89cd725b0ab7a36bb04fd4735515e5e3a20feR279-R282